llm.py

```
@@ -71,28 +75,29 @@ def get_provider():
 
-    if val == "ollama" and OllamaProvider:
+    if val == "ollama" and OllamaProvider is not None:
+        ollama_cls = cast(Any, OllamaProvider)
         endpoint = os.getenv("OLLAMA_ENDPOINT", "http://localhost:11434")
         model = os.getenv("OLLAMA_MODEL", "llama3.1:8b")
         # малый таймаут, чтобы даже при misconfig не висеть
         timeout_s = float(os.getenv("OLLAMA_TIMEOUT", "5"))
         try:
-            return OllamaProvider(endpoint=endpoint, model=model, timeout_s=timeout_s)
+            return ollama_cls(endpoint=endpoint, model=model, timeout_s=timeout_s)
         except Exception:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)